mipengine/algorithms/ttest_independent.py

```diff
@@ -49,9 +52,9 @@ def get_specification(cls):
                 ),
                 x=InputDataSpecification(
                     label="Covariate (independent)",
```

Comment:
"Variable" and "Covariate" are not relevant here. Label `y` as "Variable of interest" and `x` as "Grouping variable".